integration/query_test.go

```diff
@@ -274,7 +274,7 @@ func TestQueryCount(t *testing.T) {
 	}{
 		"CountAllDocuments": {
 			command:  bson.D{{"count", collection.Name()}},
-			response: 45,
+			response: 48,
```

Comment:
Those tests really should be moved to compat tests

Reply:
fixed